HISTORY.md

```diff
@@ -1,3 +1,10 @@
+2019-11-02 / 2.0.0
```

Comment:
You can just put "unreleased" here or similar, otherwise this date will have to keep getting updated.